modules/tinymce/src/themes/silver/test/ts/webdriver/editor/DisabledNestedMenuItemTest.ts

```diff
@@ -11,13 +11,16 @@ describe('webdriver.tinymce.themes.silver.editor.menubar.DisabledNestedMenuItemT
   const codeMenuItemSelector = '[role="menuitem"]:contains("Code")';
   const preferencesMenuItemSelector = '[title="Preferences"]';
   const servicesMenuItemSelector = '[title="Services"]';
+  const highligthedCodeMenuSelector = '.tox-mbtn--active .tox-mbtn__select-label:contains("Code")';
```

Comment:
This is still using `Code` and `Tools` so doesn't look like this was updated.

Reply:
Code is my custom menu. Tools is wrong tho. I've fixed.

Reply:
Ahh right... that's a bit confusing though :shrug: